distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -46,7 +46,7 @@ grant
 }
 
 grant
-  codeBase "file:/security-migratable" {
+  codeBase "file:/security-migratable/catalog-rest-endpoint" {
```

Comment:
❓Could we grant those permissions to a WSS4J, CXF or DDF security bundle instead?

Reply:
This is yet another example of where we need to start creating our own permissions and privileging lower-level code. For now, however, I don't imagine there's anyway around it.